static/app/components/nav/sidebar.tsx

```diff
@@ -0,0 +1,220 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import Feature from 'sentry/components/acl/feature';
+import OrganizationAvatar from 'sentry/components/avatar/organizationAvatar';
+import Link from 'sentry/components/links/link';
+import {useNavContext} from 'sentry/components/nav/context';
+import Submenu from 'sentry/components/nav/submenu';
+import {useNavIndicator} from 'sentry/components/nav/useNavIndicator';
+import {
+  isNavItemActive,
+  isNonEmptyArray,
+  isSubmenuItemActive,
+  makeLocationDescriptorFromTo,
+  type NavSidebarItem,
+  resolveNavItemTo,
+} from 'sentry/components/nav/utils';
+import {space} from 'sentry/styles/space';
+import theme from 'sentry/utils/theme';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+
+function Sidebar() {
+  const organization = useOrganization();
+
+  return (
+    <Fragment>
+      <SidebarWrapper role="navigation" aria-label="Primary Navigation">
+        <SidebarHeader>
+          <OrganizationAvatar organization={organization} size={32} />
+        </SidebarHeader>
+        <SidebarItems />
+      </SidebarWrapper>
+      <Submenu />
+    </Fragment>
+  );
+}
+
+export default Sidebar;
+
+export function SidebarItems() {
+  const nav = useNavContext();
+  return (
+    <Fragment>
+      <SidebarBody>
+        {nav.config.main.map(item => (
+          <SidebarItem key={item.label} item={item} />
+        ))}
+      </SidebarBody>
+      {isNonEmptyArray(nav.config.footer) && (
+        <SidebarFooter>
+          {nav.config.footer.map(item => (
+            <SidebarItem key={item.label} item={item} />
+          ))}
+        </SidebarFooter>
+      )}
+    </Fragment>
+  );
+}
+
+const SidebarWrapper = styled('div')`
+  height: 40px;
+  width: 100vw;
+  padding: ${space(2)} 0;
+  border-right: 1px solid ${theme.translucentGray100};
+  background: #3e2648;
+  background: linear-gradient(180deg, #3e2648 0%, #442c4e 100%);
```

Comment:
👃 for the hardcoded colors, idk if we have anything close to these in our theme / styleguide tho. would be cool if we could use something from there though.

Reply:
Agree! We don't seem to have anything remotely close, unfortunately. I added a comment in 959e39dccc9 for future ref.